setup.py

```diff
@@ -5,56 +5,76 @@
-      long_description=long_description(),
-      author='Leonid S Shestera',
-      author_email='leonid@shestera.ru',
-      maintainer='Ecometrica',
-      maintainer_email='dev@ecometrica.com',
-      url='http://github.com/ecometrica/django-multisite',
-      packages=find_packages(),
-      include_package_data=True,
-      package_data={'multisite': files},
-      install_requires=install_requires,
-      setup_requires=['pytest-runner'],
-      tests_require=['coverage', 'mock', 'pytest', 'pytest-cov',
-                     'pytest-django', 'pytest-pythonpath', 'tox'],
-      test_suite="multisite.tests",
-      classifiers=['Development Status :: 4 - Beta',
-                   'Environment :: Web Environment',
-                   'Framework :: Django',
-                   'Intended Audience :: Developers',
-                   'License :: OSI Approved :: BSD License',
-                   'Operating System :: OS Independent',
-                   'Programming Language :: Python',
-                   'Programming Language :: Python :: 2.7',
-                   'Programming Language :: Python :: 3.4',
-                   'Programming Language :: Python :: 3.5',
-                   'Programming Language :: Python :: 3.6',
-                   'Topic :: Internet',
-                   'Topic :: Internet :: WWW/HTTP',
-                   'Topic :: Software Development :: Libraries',
-                   'Topic :: Utilities'],
-      )
+setup(
+    name="django-multisite",
+    version=version["__version__"],
+    description="Serve multiple sites from a single Django application",
+    long_description=long_description(),
+    author="Leonid S Shestera",
+    author_email="leonid@shestera.ru",
+    maintainer="Ecometrica",
+    maintainer_email="dev@ecometrica.com",
+    url="http://github.com/ecometrica/django-multisite",
+    packages=find_packages(),
+    include_package_data=True,
+    package_data={"multisite": files},
+    install_requires=["tldextract>=1.2"],
+    setup_requires=["pytest-runner"],
+    tests_require=[
+        "coverage",
+        "mock",
+        "pytest",
+        "pytest-cov",
+        "pytest-django",
+        "pytest-pythonpath",
+        "tox",
+    ],
+    test_suite="multisite.tests",
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "Environment :: Web Environment",
+        "Framework :: Django",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: BSD License",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Topic :: Internet",
+        "Topic :: Internet :: WWW/HTTP",
+        "Topic :: Software Development :: Libraries",
+        "Topic :: Utilities",
+    ],
+)
```

Comment:
These cosmetic changes seem fairly unnecessary. Are they all cosmetic? Kind of hard to see in the hunk, but I assume they are. Cosmetic changes should be in a separate commit.